lib/config/presets/internal/workarounds.ts

```diff
@@ -35,6 +36,16 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  ignoreWeb3jCoreWithOldReleaseTimestamp: {
+    description: 'Ignore web3j 5.0.0 release',
+    packageRules: [
+      {
+        matchDatasources: ['maven'],
+        matchPackageNames: ['org.web3j:core'],
+        allowedVersions: '/^(?!5\\.0\\.0)/',
```

Comment:
```suggestion
        allowedVersions: '!/^5\\.0\\.0/',
```
Renovate uses `!` to invert a Regex.

Reply:
I see. Thanks for this suggestion!